autoload/utils/exec/job.vim

```diff
@@ -97,18 +88,26 @@ function! s:nVimExit(job_id, data, event) abort
         return
     endif
     call s:createQuickFix()
+    call s:closeBuffer()
     if a:data != 0
         copen
     endif
 endfunction
 
 function! s:createJobBuf() abort
-    silent execute 'belowright 10split ' . s:cmake4vim_buf
+    let l:in_quickfix = getwininfo(win_getid())[0]['quickfix']
+    if l:in_quickfix
+        silent execute 'edit ' . s:cmake4vim_buf
+    else
+        silent execute 'belowright 10split ' . s:cmake4vim_buf
+    endif
     setlocal bufhidden=hide buftype=nofile buflisted nolist
-    setlocal noswapfile nowrap nomodifiable
+    setlocal noswapfile nowrap modifiable
```

Comment:
Needed for the `job` to write to the buffer.

Reply:
Hmm, in `s:appendLine()` function I switch buffer to `modifiable` mode, I see that this function isn't used, but if you allow to modify the buffer in any time, it means that I can change this buffer and it is not good.
Can we modify the buffer only if we got some information from the job?